src/components/ContactForm/ContactForm.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ContactForm from './ContactForm';
+
+describe('ContactForm component', () => {
+  const handleSubmitMock = jest.fn();
+  const props = { onSubmit: handleSubmitMock };
+
+  const contactFormComponent = shallow(<ContactForm {...props} />);
+  const label = contactFormComponent.find('label');
+  const input = label.find('input');
+
+  it('should render with props', () => {
+    expect(contactFormComponent.exists()).toBeTruthy();
+  });
+
+  it('contains label tag', () => {
+    expect(label).toHaveLength(2);
+  });
+
+  it('contains input tag', () => {
+    expect(input).toHaveLength(2);
+  });
+
+  it('form submit', () => {
+    const contactFormElement = contactFormComponent.find('form');
+    contactFormElement.simulate('submit', { preventDefault: () => {} });
```

Comment:
please move `{ preventDefault: () => {} }` to const `event`

Reply:
const event with value '{ preventDefault: () => {} }' has been made.